.gitignore

```diff
@@ -11,4 +11,4 @@ docker/db
 build/
 
 *.csv
-tmp/
+tmp/
```

Comment:
Is there any tmp directory getting created while Sugarizer-Server setup?
If no, then .gitignore does not needs to be modified.
If yes, then add a new line after this line.